Dockerfile

```diff
@@ -19,4 +19,5 @@ RUN cd /usr/bin && \
     ln -s control-plane-operator token-minter
 
 ENTRYPOINT /usr/bin/hypershift
-LABEL io.openshift.hypershift.control-plane-operator-subcommands=true
```

Comment:
why does this need to be in this image? I can only see how it has value in the control plane image?